docs/topics/certificaterotation.md

```diff
@@ -1,60 +0,0 @@
-# Cluster certificate rotation
```

Comment:
Why are we deleting these docs?

Reply:
It has duplicated/stale information, I pushed another one with my previous PR.

https://github.com/Azure/aks-engine/blob/master/docs/topics/rotate-certs.md